frontend/src/components/Answer.jsx

```diff
@@ -1,16 +1,25 @@
 import PropTypes from "prop-types";
+import { useState } from "react";
 
 function Answer({ title, res }) {
-  return (
-    <button
-      type="submit"
-      className="answer"
-      onClick={
-        res
-          ? () => window.alert("Bonne réponse !")
-          : () => window.alert("Mauvaise réponse !")
+  const [isActive, setActive] = useState(false);
+
+  const ToggleClass = () => {
```

Comment:
functions start with a lowercase letter ;)

toggling is the action of inverting a boolean value: true becomes false, and false becomes true. your function isn't really toggling: it's always setting to true. this is more a `const activate = () => {`